src/test/java/task40/helpes/HomeMailPage.java

```diff
@@ -0,0 +1,34 @@
+package task40.helpes;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomeMailPage {
```

Comment:
Lets have a call on Monday and discuss page class
The main idea is that your page class shouldn't return webElements. You should have methods that do actions on the page

Reply:
Done